packages/react-next/src/components/SwatchColorPicker/SwatchColorPicker.base.tsx

```diff
@@ -147,7 +147,10 @@ export class SwatchColorPickerBase extends React.Component<ISwatchColorPickerPro
       />
     );
   }
-
+  private _onRenderItem = (item: IColorCellProps, index: number): JSX.Element => {
+    const { onRenderColorCell = this._renderOption } = this.props;
+    return onRenderColorCell(item, this._renderOption) as JSX.Element;
```

Comment:
Do we need `as JSX.Element` here?
I think as we added `null` to the return type of `onRenderItem` in `Grid.types.ts` , we can add the `null` return type here as well ...

Reply:
I have added null to the return type of _Grid.types.ts_ and have remove _as JSX.Element_